microsoft/validation/k_inference_js.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+import pandas as pd
+import json
+from sklearn.metrics import confusion_matrix, precision_score, recall_score, f1_score, accuracy_score
+import librosa
+from Config import Config
+from keras import losses, models, optimizers
+from keras.activations import relu, softmax
+from keras.callbacks import (EarlyStopping, LearningRateScheduler,
+                             ModelCheckpoint, TensorBoard, ReduceLROnPlateau)
+from keras.layers import (Convolution2D, GlobalAveragePooling2D, BatchNormalization, Flatten,
+                          GlobalMaxPool2D, MaxPool2D, concatenate, Activation, Input, Dense)
+from keras.utils import Sequence, to_categorical
+from keras import backend as K
+import argparse
+
+parser = argparse.ArgumentParser(

```

Comment:
we should be wrapping the executable portion of scripts in a

`if __name__ == "__main__":`